src/styles/Search.css

```
@@ -0,0 +1,26 @@
+.search-form {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  
+  }
+
+  .search-input {
+    width: 20%;
+    height: 50px;
+    border-radius: 11px;
+    font-size: 20px;
+    padding: 0 10px;
+    margin-right: 1em;
+  
+  }
```

The classnames themselves should always be at the beginning of the line:
```suggestion
.search-input {
    width: 20%;
    height: 50px;
    border-radius: 11px;
    font-size: 20px;
    padding: 0 10px;
    margin-right: 1em;
}
```